### ISSUE TITLE
[Bug] Does not work in Release mode on UWP

### ISSUE THREAD

**Issue by YZahringer** [AndreiMisiukevich/TouchEffect#67]
### Description
Effects and `Command` do not work in `Release` mode on `UWP`.

### Steps to Reproduce
1. Open `TouchEffectSample` solution
2. Select `Release` configuration
3. Right click on `TouchEffectSample.UWP` and select `Deploy`
4. Run app and check `Hover/Pressed` effects and `Command` -> No response

### Basic Information

- Version with issue: 4.1.53
- Last known good version: ?
- OS: Windows 10 build 1909

**Comment by AndreiMisiukevich** [comment]
Hi
Try this

```csharp
var assembliesToInclude = new List();
assembliesToInclude.Add(typeof(TouchEffect.UWP.PlatformTouchEff).GetTypeInfo().Assembly);
Xamarin.Forms.Forms.Init(e, assembliesToInclude);
```

**Comment by YZahringer** [comment]
@AndreiMisiukevich Thanks! it fix the issue 👍

I created PR with documentation update for UWP and small bug fix: #68
